Source/WebKit/PlatformPlayStation.cmake

```diff
@@ -160,6 +152,54 @@ list(APPEND WebKit_INCLUDE_DIRECTORIES
     "${WEBKIT_DIR}/WebProcess/WebPage/libwpe"
 )
 
+if (USE_COORDINATED_GRAPHICS)
+    list(APPEND WebKit_SOURCES
+        Shared/CoordinatedGraphics/CoordinatedGraphicsScene.cpp
+        Shared/CoordinatedGraphics/SimpleViewportController.cpp
+
+        Shared/CoordinatedGraphics/threadedcompositor/CompositingRunLoop.cpp
+        Shared/CoordinatedGraphics/threadedcompositor/ThreadedCompositor.cpp
+        Shared/CoordinatedGraphics/threadedcompositor/ThreadedDisplayRefreshMonitor.cpp
+
+        WebProcess/WebPage/CoordinatedGraphics/LayerTreeHost.cpp
+    )
+endif ()
+
+if (USE_GRAPHICS_LAYER_WC)
+    list(APPEND WebKit_SOURCES
+        GPUProcess/graphics/RemoteGraphicsContextGLWC.cpp
+
+        GPUProcess/graphics/wc/RemoteWCLayerTreeHost.cpp
+        GPUProcess/graphics/wc/WCContentBufferManager.cpp
+        GPUProcess/graphics/wc/WCScene.cpp
+        GPUProcess/graphics/wc/WCSceneContext.cpp
+
+        UIProcess/wc/DrawingAreaProxyWC.cpp
+
+        WebProcess/GPU/graphics/wc/RemoteGraphicsContextGLProxyWC.cpp
+        WebProcess/GPU/graphics/wc/RemoteWCLayerTreeHostProxy.cpp
+
+        WebProcess/WebPage/CoordinatedGraphics/LayerTreeHostTextureMapper.cpp
+
+        WebProcess/WebPage/wc/DrawingAreaWC.cpp
+        WebProcess/WebPage/wc/GraphicsLayerWC.cpp
+        WebProcess/WebPage/wc/WCLayerFactory.cpp
+        WebProcess/WebPage/wc/WCTileGrid.cpp
+    )
+
+    list(APPEND WebKit_INCLUDE_DIRECTORIES
+        "${WEBKIT_DIR}/GPUProcess/graphics/wc"
+        "${WEBKIT_DIR}/Shared/wc"
+        "${WEBKIT_DIR}/UIProcess/wc"
+        "${WEBKIT_DIR}/WebProcess/GPU/graphics/wc"
+        "${WEBKIT_DIR}/WebProcess/WebPage/wc"
+    )
+
+    list(APPEND WebKit_MESSAGES_IN_FILES
+        GPUProcess/graphics/wc/RemoteWCLayerTreeHost
+    )
+endif ()
+
```

Comment:
Do you think this part should be moved to common place somewhere?

Reply:
I think after we have both platforms up and running with WC we should find a common place for it all.